monkey/monkey_island/cc/resources/monkey.py

```diff
@@ -22,10 +21,6 @@ class Monkey(AbstractResource):
         "/api/agent/<string:guid>",
     ]
 
-    # Used by monkey. can't secure.
-    def get(self):
-        return {"config": ConfigService.format_flat_config_for_agent()}
-
```

Comment:
Why are we only removing the GET endpoint and not POST?

Reply:
The `POST` endpoint is what the agent uses to register itself with the Island. Take a look at the TODO in that method and the code.